mappings/net/minecraft/entity/ai/pathing/PathNodeType.mapping

```diff
@@ -0,0 +1,20 @@
+CLASS none/azz net/minecraft/entity/ai/pathing/PathNodeType
+	FIELD a BLOCKED Lnone/azz;
+	FIELD b AIR Lnone/azz;
+	FIELD c NORMAL Lnone/azz;
+	FIELD d TRAPDOOR Lnone/azz;
+	FIELD e FENCE Lnone/azz;
+	FIELD f LAVA Lnone/azz;
+	FIELD g WATER Lnone/azz;
+	FIELD h RAIL Lnone/azz;
+	FIELD i FIRE_NEAR Lnone/azz;
+	FIELD j FIRE Lnone/azz;
+	FIELD k CACTUS_NEAR Lnone/azz;
+	FIELD l CACTUS Lnone/azz;
+	FIELD o DOOR_OPEN Lnone/azz;
+	FIELD p DOOR_WOOD Lnone/azz;
+	FIELD q DOOR_METAL Lnone/azz;
```

Comment:
Actually no, villagers can't open the metal doors. So it's unopenable and unbreakable.